app/assets/javascripts/privileges.js

```diff
@@ -25,9 +26,12 @@ $(() => {
     const $td = $tgt.parent();
     const $input = $td.find('.js-privilege-edit');
     const name = $input.data('name');
+    const type = $input.data('type');
     const value = $input.val();
 
-    const resp = await fetch(`/admin/privileges/${name}`, {
+    if(!value && value !== 0 && value !== "0") { value = null }
```

Comment:
This line doesn't do what you think it does. `0` is already falsy so would already be caught by `!value`. You're also assigning to `value`, which is going to cause a runtime error because `value` is a `const` - you want `let value` on the original definition if you want to reassign to it.

Aside from that, not quite sure what the purpose of this line is - what are you trying to do here? I suspect there's almost certainly an easier way to express it.

Reply:
Fixed.